diff --git a/CMakeLists.txt b/CMakeLists.txt
index 20f91f8..30b5fbb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,10 +47,10 @@ target_compile_options(memsed PRIVATE -fdata-sections -ffunction-sections -fno-m
 
 # Python (needed for GLAD and dear_bindings)
 find_package(Python COMPONENTS Interpreter REQUIRED)
-execute_process(COMMAND "${Python_EXECUTABLE}" -m venv "${VENV_DIR}")
-set(Python_EXECUTABLE "${VENV_DIR}/bin/python")
-execute_process(COMMAND "${Python_EXECUTABLE}" -m pip install -U -r "${MEMSED_DIR}/requirements.txt")
-configure_file("${MEMSED_DIR}/requirements.txt" "${VENV_DIR}/.installed")
+# execute_process(COMMAND "${Python_EXECUTABLE}" -m venv "${VENV_DIR}")
+# set(Python_EXECUTABLE "${VENV_DIR}/bin/python")
+# execute_process(COMMAND "${Python_EXECUTABLE}" -m pip install -U -r "${MEMSED_DIR}/requirements.txt")
+# configure_file("${MEMSED_DIR}/requirements.txt" "${VENV_DIR}/.installed")
 
 # OpenGL (via GLAD loader)
 add_subdirectory("${GLAD_GEN}/cmake")
